frontend/lib/src/components/shared/Icon/DynamicIcon.tsx

```diff
@@ -38,32 +38,35 @@ function parseIconPackEntry(iconName: string): IconPackEntry {
   return { pack: iconPack, icon: iconNameInPack }
 }
 
-export interface DynamicIconProps {
```

Comment:
> its still two different components ... combining it will make it more complicated later if we add something that's only relevant to the DynamicIconProps level or the other way around

I hadn't considered that case 😅 Thanks for calling it out. Fixed!